binding/binding.go

```diff
@@ -144,6 +266,12 @@ func (b *Binding) Marshal() (data []byte, err error) {
 	if err = amqp.WriteShortstr(buf, b.RoutingKey); err != nil {
 		return nil, err
 	}
+	// Since marshalling is used for storage only, we can
```

Comment:
Looks like right way, but can you add params protoVersion anyway? Cause it is common pattern for marshaling storage entities (for example queue, exchange or message)

Reply:
Sure, I can add this as well